mlflow/server/auth/db/migrations/versions/8606fa83a998_initial_migration.py

```diff
@@ -0,0 +1,64 @@
+"""initial_migration
+
+Revision ID: 8606fa83a998
+Revises: 
+Create Date: 2023-07-07 23:30:50.921970
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "8606fa83a998"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
I think we can remove these comments from the migration scripts

Reply:
Agree